frontend/src/tests/utils/useBackend.test.js

```diff
@@ -42,27 +37,25 @@ describe("utils/useBackend tests", () => {
 
             var axiosMock = new AxiosMockAdapter(axios);
 
-            axiosMock.onGet("/api/admin/users").reply(404, {});
-
-            const restoreConsole = mockConsole();
+            axiosMock.onGet("/api/admin/users").reply(404, { message: "Error: Request failed with status code 404" });
 
             const { result, waitFor } = renderHook(() => useBackend(
                 ["/api/admin/users"],
+                // Stryker disable next-line StringLiteral : "GET" is the default, so removing it is equivalent
```

Comment:
Does this line have any effect in a test file? It seems odd to be disabling lines in a test and not in the actual source code.

Reply:
good point.. I think I was doing a global search for 'GET' and putting it everywhere I found it.  I'll remove it here.

Reply:
removed